web/src/pages/Settings/Streams/StreamModal/StreamModal.jsx

```diff
@@ -218,7 +216,11 @@ const StreamModal = ({ stream, onClose, apply, del }) => {
         "name": "name",
         "title": "Name",
         "type": "text",
-        "desc": NAME_DESC,
+        "desc": type == "spotify" ?
+            "This name requires a minimum of two non-special characters, and cannot have certain characters such as underscores or slashes - it will be used to select this stream from the source selection dropdown"
+            :
+            "This name can be anything - it will be used to select this stream from the source selection dropdown"
+        ,
```

Comment:
Adding a special case like this seems wrong, I think the Spotify stream should be changed to re-format an invalid name to something valid.

Reply:
I think this is going to need a special case either way, given that "This name can be anything! Oops, not that though." is a bad experience
We should totally still have backend name validation (as suggested in the new issue #986), this PR is just for the accuracy of the description field and I think that's all we strictly need right now to obviate the issue that users could come across

Reply:
I agree with @klay2000